Code/Mantid/Framework/DataHandling/src/LoadMcStas.cpp

```diff
@@ -61,6 +61,8 @@ void LoadMcStas::init() {
   declareProperty(new WorkspaceProperty<Workspace>("OutputWorkspace", "",
                                                    Direction::Output),
                   "An output workspace.");
+  // added to allow control of errorbars
+  declareProperty("ErrorBarsSetTo1",false);                
```

Comment:
Add description for this property, e.g. "When this property set to false errors set equal to data values, and when set to true all errors are set to one. Default to false"